plugins/processors/awsappsignals/processor.go

```diff
@@ -0,0 +1,268 @@
+		dps := m.Histogram().DataPoints()
+		for i := 0; i < dps.Len(); i++ {
+			for _, Mutator := range ap.metricMutators {
+				err := Mutator.Process(dps.At(i).Attributes(), resourceAttribes, false)
+				if err != nil {
+					ap.logger.Debug(failedToProcessAttribute, zap.Error(err))
+				}
+			}
+		}
+		dps.RemoveIf(func(d pmetric.HistogramDataPoint) bool {
+			for _, Mutator := range ap.customAllowlistMutators {
+				shouldBeDropped, err := Mutator.ShouldBeDropped(d.Attributes(), resourceAttribes)
+				if err != nil {
+					ap.logger.Debug(failedToProcessAttributeWithCustomRule, zap.Error(err))
+					return true
+				} else if shouldBeDropped {
+					return true
+				}
+			}
+			return false
+		})
+		for i := 0; i < dps.Len(); i++ {
+			err := ap.customReplacer.Process(dps.At(i).Attributes(), resourceAttribes, false)
+			if err != nil {
+				ap.logger.Debug(failedToProcessAttribute, zap.Error(err))
+			}
+		}
+	case pmetric.MetricTypeExponentialHistogram:
+		dps := m.ExponentialHistogram().DataPoints()
+		for i := 0; i < dps.Len(); i++ {
+			for _, Mutator := range ap.metricMutators {
+				err := Mutator.Process(dps.At(i).Attributes(), resourceAttribes, false)
+				if err != nil {
+					ap.logger.Debug(failedToProcessAttribute, zap.Error(err))
+				}
+			}
+		}
+		dps.RemoveIf(func(d pmetric.ExponentialHistogramDataPoint) bool {
+			for _, Mutator := range ap.customAllowlistMutators {
+				shouldBeDropped, err := Mutator.ShouldBeDropped(d.Attributes(), resourceAttribes)
+				if err != nil {
+					ap.logger.Debug(failedToProcessAttributeWithCustomRule, zap.Error(err))
+					return true
+				} else if shouldBeDropped {
+					return true
+				}
+			}
+			return false
+		})
+		for i := 0; i < dps.Len(); i++ {
+			err := ap.customReplacer.Process(dps.At(i).Attributes(), resourceAttribes, false)
+			if err != nil {
+				ap.logger.Debug(failedToProcessAttribute, zap.Error(err))
+			}
+		}
+	case pmetric.MetricTypeSummary:
+		dps := m.Summary().DataPoints()
+		for i := 0; i < dps.Len(); i++ {
+			for _, Mutator := range ap.metricMutators {
+				err := Mutator.Process(dps.At(i).Attributes(), resourceAttribes, false)
+				if err != nil {
+					ap.logger.Debug(failedToProcessAttribute, zap.Error(err))
+				}
+			}
+		}
+		dps.RemoveIf(func(d pmetric.SummaryDataPoint) bool {
+			for _, Mutator := range ap.customAllowlistMutators {
+				shouldBeDropped, err := Mutator.ShouldBeDropped(d.Attributes(), resourceAttribes)
+				if err != nil {
+					ap.logger.Debug(failedToProcessAttributeWithCustomRule, zap.Error(err))
+					return true
+				} else if shouldBeDropped {
+					return true
+				}
+			}
+			return false
+		})
+		for i := 0; i < dps.Len(); i++ {
+			err := ap.customReplacer.Process(dps.At(i).Attributes(), resourceAttribes, false)
+			if err != nil {
+				ap.logger.Debug(failedToProcessAttribute, zap.Error(err))
+			}
+		}
+	default:
+		ap.logger.Info("Ignore unknown metric type", zap.String("type", m.Type().String()))
```

Comment:
nit: Should this be an info level log? It seems more like either debug or warn depending on if you want to surface it.

Reply:
ack. changing to debug